components/header/index.styl

```diff
@@ -1,56 +1,42 @@
 .header
-  height 350px
-  position relative
-  background-image url('./images/Banner_3.jpg')
-  background-position center
-  background-size 526px
-
-.header nav
   width 100%
-  height 62px
-  // top 0
-  // left 0
-  border-bottom 1px solid white
-
-.is-desktop-header-icon
-  display none
-
-.header svg
-  padding 10px
-  polygon, path
-    fill white
-
-.nav-icons
-  float right
-  position relative
-  top 50%
-  transform translateY(-50%)
-
-.header-language-toggle
-  float right
-  margin 0 15px
-  position relative
-  top 50%
-  transform translateY(-50%)
+  height 100vh
+  background-color black
+  background-image url('./images/Banner_3.jpg')
+  background-position center center
+  // Not the most performant thing if we're gonna do anything fancy with it:
+  background-size cover
 
-.header-chinese
-  color white
-  text-align center
-  display inline-block
-  font-size 20px
-  margin-left 10px
+.header-content
+  height 100%
+  display flex
+  flex-direction column
+  justify-content space-around
 
 .header-text
-  margin 20px
-  font-size 17px
-  line-height 1.5
-  position absolute
-  top 50%
-  transform: translate(0, -50%)
-
-.header-text h1
-  margin 30px 0
+  width 75%
+  margin 5% auto
   color white
-
-.is-desktop-header
-  display none
\ No newline at end of file
+  avant-garde-size('xl-headline')
+  font-size 50px
+  > p
+    margin 40px 0
+    &:first-child
+      margin-top 0
+    &:last-child
+      margin-bottom 0
+
+.header-section-nav
```

Comment:
@xtina-starr damons naming game is also v strong. Generic classes like this that describe the interface are easiest to understand and maintain.